test/osbuildtest.py

```diff
@@ -34,4 +34,4 @@ def run_osbuild(self, pipeline):
             raise e from None
 
         result = json.loads(r.stdout)
-        return result["tree_id"], result["output_id"]
+        return f"{self.store}/refs/{result['tree_id']}", f"{self.store}/refs/{result['output_id']}"
```

Comment:
Hm, I'm not 100% sure about this. I quite liked the current outputs, as they were about the semantics of what we produced, and could potentially be used for other things than just dereferencing the results.

Reply:
Agreed. This is only test code, but I liked the symmetry of it returning something similar to what `osbuild --json` on the command line.

Is constructing the part inconvenient or impossible somewhere?

Reply:
I liked the fact that subclasses of osbuildtest.TestCase don't need to know anything about `self.store`. But yeah, it makes sense to return the ids, I will make a change.